src/Group.cpp

```diff
@@ -102,15 +102,12 @@ Group::Group(const RString& sDir, const RString& sGroupDirName, bool bFromProfil
             RString sValue = "";
             ini.GetValue("Group", "SyncOffset", sValue);
             Trim(sValue);
-            if (!sValue.empty()) {
-                if (sValue == "NULL") {
-                    m_fSyncOffset = 0.0f;
-                } else if (sValue == "ITG") {
-                    m_fSyncOffset = -0.009f;
-                } else {
-                    LOG->Warn("Group::Group: Invalid SyncOffset value: %s in Pack.ini. Valid values are NULL and ITG. Defaulting to NULL.", sValue.c_str());
-                }
-            }
+			if (sValue == "ITG") {
+				m_fSyncOffset = -0.009f;
+			}
+			else if (sValue != "NULL" && !sValue.empty()) {
```

Comment:
If there's no SyncOffset set in the Pack.ini we should fallback onto the DefaultSyncOffset preference